packages/e2e-utils/src/mocks/backendServer.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
I think those errors poped up when ESLint got the type-information from typescript. But not sure.

Reply:
Seems to be somehow globally imported from `typescript/lib/lib.dom.d.ts`